hack/update-generated-bindata.sh

```diff
@@ -32,7 +32,8 @@ pushd "${OS_ROOT}" > /dev/null
     examples/heapster/... \
     examples/prometheus/... \
     examples/service-catalog/... \
-    pkg/image/admission/imagepolicy/api/v1/...
+    pkg/image/admission/imagepolicy/api/v1/... \
+    test/testdata/templateservicebroker/...
```

Comment:
is testdata really the final home for this?  is this just for testing?  I was under the impression it was more fundamental.